Unfortunately, this is failing when I try to build it locally:

```
[ 84%] Linking C shared library ../lib32/libphobos2-ldc-shared.so
cd /media/joseph/hexxat/code/snap/ldc2.snap/parts/ldc/build/runtime && /usr/bin/cmake -E cmake_link_script CMakeFiles/phobos2-ldc-shared_32.dir/link.txt --verbose=1
/usr/bin/cc  -fPIC -DHAVE_UNISTD_H -O3 -DNDEBUG -Wa,-mrelax-relocations=no -m32  -shared -Wl,-soname,libphobos2-ldc-shared.so.79 -o ../lib32/libphobos2-ldc-shared.so.2.0.79 ............ ../lib32/libdruntime-ldc-shared.so.2.0.79 -lm -lpthread -lrt -ldl -Wl,-rpath,/media/joseph/hexxat/code/snap/ldc2.snap/parts/ldc/build/lib32: 
objects_32/std/format.o: file not recognised: File truncated
collect2: error: ld returned 1 exit status
runtime/CMakeFiles/phobos2-ldc-shared_32.dir/build.make:1943: recipe for target 'lib32/libphobos2-ldc-shared.so.2.0.79' failed
make[2]: *** [lib32/libphobos2-ldc-shared.so.2.0.79] Error 1
```

... any thoughts on why, anyone?  This is the first time I've encountered a problem doing a multilib build on a 64-bit system (the only problems before were on 32-bit).

--- a/ldc-config/ldc2.conf
+++ b/ldc-config/ldc2.conf
@@ -10,6 +10,9 @@ default:
         "-I%%ldcbinarypath%%/../include/d/ldc",
         "-I%%ldcbinarypath%%/../include/d",
         "-L-L%%ldcbinarypath%%/../lib",
+        "-L-L%%ldcbinarypath%%/../lib32",
+        "-L-L%%ldcbinarypath%%/../lib64",
+        "-L--no-warn-search-mismatch",
         "-defaultlib=phobos2-ldc,druntime-ldc",
         "-debuglib=phobos2-ldc-debug,druntime-ldc-debug"
     ];

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -40,12 +40,14 @@ parts:
     - -DLLVM_ROOT_DIR=../../llvm/install
     - -DBUILD_LTO_LIBS=ON
     - -DLDC_INSTALL_LTOPLUGIN=ON
+    - -DMULTILIB=ON
     - -DCMAKE_VERBOSE_MAKEFILE=1
     install: ctest --output-on-failure --verbose -E "std\.net\.curl|std\.process|lit-tests"
     stage:
     - -etc/ldc2.conf
     build-packages:
-    - gcc
+    - gcc-multilib
+    - g++-multilib
     - libedit-dev
     - zlib1g-dev
     after:
@@ -66,6 +68,7 @@ parts:
     configflags:
     - -DCMAKE_BUILD_TYPE=Release
     - -DLLVM_ROOT_DIR=../../llvm/install
+    - -DMULTILIB=ON
     stage:
     - -*
     prime:

